src/openrct2/scenario/ScenarioRepository.cpp

```diff
@@ -155,6 +155,15 @@ class ScenarioFileIndex final : public FileIndex<ScenarioIndexEntry>
     void Serialise(DataSerialiser& ds, const ScenarioIndexEntry& item) const override
     {
         ds << item.Path;
+        if (ds.IsLoading())
```

Comment:
Are we sure this is the only place we need to do this

Reply:
Only place affected by the PR